.github/workflows/datastore-emulator-system-tests.yaml

```diff
@@ -0,0 +1,35 @@
+on:
```

Comment:
Have you verified that the firestore system tests work with the emulator? We shouldn't add this until that is verified and fixed if need be. Same for Datastore.

Reply:
Yes, I ran Firestore and Datastore tests with emulators, they all pass. The only thing I forgot is exclude Firestore smoke test.